probatus/feature_elimination/feature_elimination.py

```diff
@@ -401,19 +451,35 @@ def fit(
             else:
                 current_model = clone(self.model)
 
-            # Perform CV to estimate feature importance with SHAP
-            results_per_fold = Parallel(n_jobs=self.n_jobs)(
-                delayed(self._get_feature_shap_values_per_fold)(
-                    X=current_X,
-                    y=self.y,
-                    model=current_model,
-                    train_index=train_index,
-                    val_index=val_index,
-                    sample_weight=sample_weight,
-                    **shap_kwargs,
+            # Early stopping enabled (or not)
+            if not (self.early_stopping_rounds and self.eval_metric):
```

Comment:
Would be nice to warn the user if they provide early_stopping_rounds, but not eval_metric (and vice-versa), that early stopping won't be used.

Reply:
agreed, thanks, great addition